Rakefile

```diff
@@ -65,7 +70,8 @@ namespace :build do
     transform = RUBY_PLATFORM =~ /darwin/ ? "-s " : "--transform s"
     transform << "',^,#{tar_basename}/,'"
 
-    `tar -C ../manageiq #{transform} -X #{exclude_file} -hcvzf #{tarball} .`
+    # Everything from */tmp/* should be excluded, except for tmp/cache/sti_loader.yml
+    `tar -C ../manageiq #{transform} --exclude-tag='cache/sti_loader.yml' -X #{exclude_file} -hcvzf #{tarball} .`
```

Comment:
That's what I'm trying to achieve - only keep tmp/cache/sti_loader.yml.

Using exclude-tag, it will remove everything from the directory that contains the file specified, with the exception of the actual file.

Since what's specified in exclude-tag (cache/sti_loader.yml) is under 'tmp', this removes everything from tmp but keeping tmp/cache/sti_loader.yml.